pkg/cmd/pr/status/http.go

```diff
@@ -47,17 +49,30 @@ func pullRequestStatus(httpClient *http.Client, repo ghrepo.Interface, options r
 		ReviewRequested edges
 	}
 
+	// SPIKE: Let's see if we can detect here and wire through
```

Comment:
We could also detect higher up and pass down the result of that. It would go hand in hand with injecting a mock detector into the `StatusOptions` struct as mentioned in another comment in the tests.

Reply:
If we push the detection up higher and add a new field as I suggested in another comment we can copy what is happening in the `issue list` command. Basically do the feature detection and pass through the appropriate fields to retrieve based on the features.

https://github.com/cli/cli/blob/1081e51648da4dba148feb33af3d18c4d43f3bbc/pkg/cmd/issue/list/list.go#L150